paddle/fluid/framework/dim.h

```diff
@@ -57,32 +53,6 @@ class Dim : public Array<int64_t, D> {
   HOST std::string to_string() const;
 };
 
-namespace detail {
-template <int kStart, int kEnd, bool kStop>
-struct FortranOrderIndexingConstructorFunctor {
-  HOSTDEVICE inline static void Run(const int64_t* in, int64_t* idx,
-                                    int64_t* out) {
-    out[kStart] = (*idx) % in[kStart];
-    (*idx) /= in[kStart];
-    FortranOrderIndexingConstructorFunctor<kStart + 1, kEnd,
-                                           kStart + 1 == kEnd>::Run(in, idx,
-                                                                    out);
-  }
-};
-
-template <int kStart, int kEnd>
-struct FortranOrderIndexingConstructorFunctor<kStart, kEnd, true> {
-  HOSTDEVICE inline static void Run(const int64_t* in, int64_t* idx,
-                                    int64_t* out) {}
-};
-}  // namespace detail
-
-template <int D>
-HOSTDEVICE Dim<D>::Dim(int64_t idx, const Dim<D>& size) {
-  detail::FortranOrderIndexingConstructorFunctor<0, D, D == 0>::Run(
-      size.Get(), &idx, this->GetMutable());
-}
-
 template <int idx, int D>
 HOSTDEVICE inline int64_t get(const Dim<D>& dim) {
```

Comment:
Also remove `get`. I think that nobody would prefer `get` rather than `operator []`.